### PR TITLE
[google_maps_flutter_web ] Update README.md to discuss mouse issue when stacked

### PR DESCRIPTION
**Pull request by idkq**
When stacked, issues with mouse occur. This need to be documented.

This documents this issue and its limitation https://github.com/flutter/flutter/issues/73830

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran [the auto-formatter]. (Unlike the flutter/flutter repo, the flutter/plugins repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [x] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[relevant style guides]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[exempt from version changes]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates
[following repository CHANGELOG style]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style
[the auto-formatter]: https://github.com/flutter/plugins/blob/master/script/tool/README.md#format-code
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests

### CODE REVIEW

File: packages/google_maps_flutter/google_maps_flutter_web/README.md [hunk 1]
```diff
@@ -46,3 +46,5 @@ There's no `defaultMarkerWithHue` in web. If you need colored pins/markers, you
 Indoor and building layers are still not available on the web. Traffic is.
 
 Only Android supports "[Lite Mode](https://developers.google.com/maps/documentation/android-sdk/lite)", so the `liteModeEnabled` constructor argument can't be set to `true` on web apps.
+
+When stacked below other widgets, pointer_interceptor libary must be used to capture mouse events. See issue https://github.com/flutter/flutter/issues/73830
```
Comment (ditman) [inline packages/google_maps_flutter/google_maps_flutter_web/README.md:50]:
```suggestion
Google Maps web uses `HtmlElementView` to render maps. When a `GoogleMap` is stacked below other widgets, [`package:pointer_interceptor`](https://www.pub.dev/packages/pointer_interceptor) must be used to capture mouse events on the Flutter overlays. See issue [#73830](https://github.com/flutter/flutter/issues/73830)
```

**Review by ditman — DISMISSED** [review]
Thanks @idkq for your contribution!

Unfortunately this can't be merged as is; can you please follow the instructions in the check-list in the description of the PR so this becomes mergeable?

Thanks!

**Comment by idkq** [comment]
This documents this issue and its limitation https://github.com/flutter/flutter/issues/73830

**Comment by cedvdb** [comment]
thank you for this. I did not know there was a solution to this

**Comment by stuartmorgan** [comment]
@idkq Are you planning on addressing the remaining items in the checklist so we can move forward with this?

**Comment by idkq** [comment]
@stuartmorgan Hi apologies for the delay, what else is missing? Can you confirm this is exempt of changelog and pubspec changes since it is a readme only?

**Comment by ditman** [comment]
I've updated the PR slightly to conform with the changelog style ("_Updates `README.md`_" instead of "_Update the `README.md`_"), and make it publishable by bumping this a patch version.

This way, the changes to the README will be published immediately to pub.dev.

PTAL @stuartmorgan!

File: packages/google_maps_flutter/google_maps_flutter_web/README.md [hunk 2]
```diff
@@ -46,3 +46,5 @@ There's no `defaultMarkerWithHue` in web. If you need colored pins/markers, you
 Indoor and building layers are still not available on the web. Traffic is.
 
 Only Android supports "[Lite Mode](https://developers.google.com/maps/documentation/android-sdk/lite)", so the `liteModeEnabled` constructor argument can't be set to `true` on web apps.
+
+Google Maps web uses `HtmlElementView` to render maps. When a `GoogleMap` is stacked below other widgets, [`package:pointer_interceptor`](https://www.pub.dev/packages/pointer_interceptor) must be used to capture mouse events on the Flutter overlays. See issue [#73830](https://github.com/flutter/flutter/issues/73830)
```
Comment (stuartmorgan) [inline packages/google_maps_flutter/google_maps_flutter_web/README.md:50]: This should probably be "Google Maps for web".

Comment (stuartmorgan) [inline packages/google_maps_flutter/google_maps_flutter_web/README.md:50]: The last sentence is missing a period.

**Review by stuartmorgan — CHANGES_REQUESTED** [review]

File: packages/google_maps_flutter/google_maps_flutter_web/README.md (hunk 2 above)
Comment (ditman) [reply packages/google_maps_flutter/google_maps_flutter_web/README.md]: Indeed. Fixed!

Comment (ditman) [reply packages/google_maps_flutter/google_maps_flutter_web/README.md]: It doesn't now!

**Review by stuartmorgan — APPROVED** [review]
LGTM

**Review by stuartmorgan — APPROVED** [review]

**Comment by stuartmorgan** [comment]
@ditman Oops, you'll need to approve this as well since you aren't the original author, which causes it to require two approvals. (It's fine to "self"-approve in this context since I've approved the parts you've written, and your changes conceptually only need one approval.)

**Review by ditman — APPROVED** [review]

**Comment by idkq** [comment]
Thank you all

**Comment by ditman** [comment]
@idkq thanks for the contribution again! Do keep them coming!

**Pull request merged**